node/src/main/scala/coop/rchain/node/balance/ValidatorPerValidatorMap.scala

```diff
@@ -0,0 +1,90 @@
+package coop.rchain.node.balance
+
+import coop.rchain.casper.protocol.BlockMessage
+import coop.rchain.crypto.codec.Base16
+
+object ValidatorPerValidatorMap {
```

Comment:
Is it supposed to be called VaultPerValidator? Or PerValidatorVaults? it is pubkey to REV address map, right?

Reply:
Which name do you prefer? 
> it is pubkey to REV address map, right?
yes. Validator pub key to one per validator vault address.

Reply:
I think it should be `PerValidatorVaults` as its a name we use for such vaults